### PR TITLE
Shl doc explicate log2 for PerformSegmentation and NormalizeSomaticReadCounts

### PR DESCRIPTION
**Pull request by sooheelee**
@samuelklee's request described in #3156.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java [hunk 1]
```diff
@@ -48,6 +48,10 @@
  *   --output tumor.seg
  * </pre>
  *
+ * While the tangent normalized (tn) data are log2 transformed, as indicated by the --log2Input true option,
```
Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: Perhaps: "While the tangent normalized (tn) input coverage may be log2 transformed, if indicated by the --log2Input true option..."

That is, PerformSegmentation will still work if you give non-log2 input (produced by some other tool besides NormalizeSomaticReadCounts) and specify --log2Input false, and the output will again be non-log2.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java [hunk 2]
```diff
@@ -48,6 +48,10 @@
  *   --output tumor.seg
  * </pre>
  *
+ * While the tangent normalized (tn) data are log2 transformed, as indicated by the --log2Input true option, 
+ * the resulting segment means in the .seg file are not. Currently, NormalizeSomaticReadCounts can only produce
```
Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: Could add Javadoc link to NormalizeSomaticReadCounts.

**Comment by samuelklee** [comment]
@droazen @sooheelee Let's get this doc change into the beta.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: I avoid use of "input" "output" in documentation. Also, this description refers specifically to what is happening in the two example commands immediately above. So, how about
```
 * In these example commands, while the tangent normalized (tn) coverage data are log2 transformed, as indicated by the --log2Input true option, 
 * the resulting segment means in the .seg file are not. PerformSegmentation accepts either non-log2 (specify --log2Input false) and log2 data.
 * However, {@link NormalizeSomaticReadCounts}, which calculates pretangent and tangent normalized coverage, can only produce log2-transformed counts.
```

File: src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java (hunk 2 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: Yup.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: No wait, hold on.

Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java]: How about:
```
* {@link NormalizeSomaticReadCounts}, which calculates pretangent and tangent normalized coverage, can only produce log2-transformed counts.
 * These example commands show the tangent normalized (tn) coverage data are log2 transformed with the --log2Input true option.
 * PerformSegmentation accepts either non-log2 (specify --log2Input false) or log2 data and for either case gives segment means as non-log2 values.
```

**Comment by sooheelee** [comment]
@samuelklee Expanded on your feedback. Let me know if the changes are okay.

**Comment by sooheelee** [comment]
Based on DR's email, I rebased these commits to get the travis tests going.

**Comment by samuelklee** [comment]
👍 Thanks!

**Pull request merged**